2-mandatory/3-groceries.js

```diff
@@ -43,13 +62,13 @@ Exercise 3:
 */
 // Gather daily item counts into this object
 let numberOfItemsPerWeek = {
-  monday: 0,
-  tuesday: 0,
-  wednesday: 0,
-  thursday: 0,
-  friday: 0,
-  saturday: 0,
-  sunday: 0,
+  monday: weeklyMealPlan.monday.length,
+  tuesday: weeklyMealPlan.tuesday.length,
+  wednesday: weeklyMealPlan.wednesday.length,
+  thursday: weeklyMealPlan.thursday.length,
+  friday: weeklyMealPlan.friday.length,
+  saturday: weeklyMealPlan.saturday.length,
+  sunday: weeklyMealPlan.sunday.length,
```

Comment:
this produces the correct result, but how can you achieve the same result by looping through `weeklyMealPlan`?